e2e/e2eutils/setup.go

```diff
@@ -105,7 +105,11 @@ func MakeDeployParams(t require.TestingT, tp *TestParams) *DeployParams {
 		ColosseumSegmentsLengths:       "2,2,3,3",
 
 		SecurityCouncilNumConfirmationRequired: 1,
-		SecurityCouncilOwners:                  []common.Address{addresses.Challenger1, addresses.Alice, addresses.Bob, addresses.Mallory},
+		SecurityCouncilOwners: []common.Address{
+			common.HexToAddress("0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65"),
+			common.HexToAddress("0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc"),
+			common.HexToAddress("0x90F79bf6EB2c4f870365E785982E1f101E93b906"),
```

Comment:
Which address are these?

Reply:
Don't we need to change also in `e2e/setup.go`?

Reply:
Why not use an existing address variables?